guides/drain-safe-app/01-bootstrap-the-app.md

```diff
@@ -55,6 +55,9 @@ On this page, you should see a card with a button to add a custom app. Click on
 ![Screenshot of add custom app modal](/guides/drain-safe-app/images/add-custom-app.png)
 
 Click "add", and it should open the app. You'll also find it in the app list from now on
+
+**Tip**: When using the Brave Browser this only works when the "Shields" feature is deactivated since `localhost:3000/manifest.json` is dedecated as a tracker.
```

Comment:
Thanks for the contribution! 

> since `localhost:3000/manifest.json` is dedecated as a tracker.

This phrase is a little bit incorrect. Brave doesn't allow any localhost connections when shields are UP; source: https://github.com/brave/brave-browser/issues/11733. Could you please change the phrasing, and then we'll be happy to merge this? 🙏

Reply:
Hi @mikheevm, will change the phrasing in a minute. 

One question: What do you mean by
> `localhost:3000/manifest.json` is dedecated as a tracker.

Is there anything I can read up on?

Reply:
sorry, I somehow thought `manifest.json` for safe is deprecated. all good, thanks 🙇